app/views/ui_ansible_roles/show.json.rabl

```diff
@@ -1,3 +1,18 @@
 object @ansible_role
 
-extends "api/v2/ansible_roles/show"
+attributes :id, :name
+code :hostgroupsCount do |role|
```

Comment:
I don't think we use camelCase elsewhere in our APIs. We do have at least one example of snake_case specifically for counts:
https://github.com/theforeman/foreman/blob/b9c2dd00bed537b73bbad5ba529130e8717dfabb/app/views/api/v2/smart_proxies/main.json.rabl#L5

Reply:
First of all, thank you very much for the swift feedback!

Changed to snake_case. Force of JS habit...